modules/5_install/install.tf

```diff
@@ -448,3 +448,13 @@ resource "null_resource" "csi_driver_config" {
     ]
   }
 }
+
+resource "ibm_pi_operations" "pi-my-ins" {
```

Comment:
Please follow the naming conventions for resource naming. Replace `-` with `_`.
Also give a good name.. eg: "fips_bastion_reboot".